Mathlib/CategoryTheory/SmallObject/TransfiniteCompositionLifting.lean

```diff
@@ -168,7 +168,8 @@ noncomputable def lift : (sqFunctor c p f g).obj (Opposite.op j) where
   w₂ := (F.isColimitOfIsWellOrderContinuous j hj).hom_ext (fun ⟨i, hij⟩ ↦ by
     have := (s.1 ⟨i, hij⟩).w₂
     dsimp at this ⊢
-    rw [liftHom_fac_assoc _ _ _ hij, this, Cocone.w_assoc])
+    rw [liftHom_fac_assoc _ _ _ hij, this, Cocone.w_assoc]
+    rfl)
```

Comment:
The goal state here is `c.ι.app i ≫ g = c.ι.app ((Set.principalSegIio j).toRelEmbedding ⟨i, hij⟩) ≫ g`, which does look like there is a `simp` lemma missing here.

Reply:
Thanks for the suggestion!